cmd/crank/render/cmd.go

```diff
@@ -42,6 +42,7 @@ type Cmd struct {
 	CompositeResource string `arg:"" help:"A YAML file specifying the composite resource (XR) to render."                                        type:"existingfile"`
 	Composition       string `arg:"" help:"A YAML file specifying the Composition to use to render the XR. Must be mode: Pipeline."              type:"existingfile"`
 	Functions         string `arg:"" help:"A YAML file or directory of YAML files specifying the Composition Functions to use to render the XR." type:"path"`
+	FunctionSecrets   string `arg:"" help:"A YAML file or directory of YAML files specifying the Secrets to use for Functions to render the XR." type:"secrets"`
```

Comment:
Probably better to have it as a flag rather than an argument, I'd expect a small number of functions to leverage it.

Reply:
Oh right, make sense. Will fix it.